src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -131,49 +134,64 @@ object GermlineAssemblyCaller {
                 referenceEnd
               )
 
-            val referenceContig = reference.getContig(referenceName)
+            val referenceContig = reference.getContig(contigName)
 
             val regionReads = window.currentRegions()
             // Find the reads the overlap the center locus/ current locus
             val currentLocusReads =
               regionReads
                 .filter(_.overlapsLocus(window.currentLocus))
 
-            val pileup = Pileup(currentLocusReads, referenceName, window.currentLocus, referenceContig)
```

Comment:
example where more lines is cleaner imho: having to count 3 or 4 args across a line is much harder than scanning down per-arg lines.